lib/src/main/java/com/orange/ods/compose/component/navigationdrawer/OdsModalDrawer.kt

```diff
@@ -165,12 +166,13 @@ object OdsModalDrawer {
                 OdsListItem(
                     text = text,
                     textColor = if (selected) OdsTheme.colors.primaryVariant else OdsTheme.colors.onSurface,
-                    textStyle = if (selected) OdsTheme.typography.titleS else OdsTheme.typography.titleS.copy(fontWeight = FontWeight.Bold),
+                    textStyle = OdsTextStyle.TitleS,
                     modifier = modifier
                         .selectable(selected = selected, onClick = { onClick(this@ListItem) })
                         .let {
                             if (selected) it.background(OdsTheme.colors.primaryVariant.copy(alpha = SelectedItemOpacity)) else it
                         },
+                    textFontWeight = if (selected) null else FontWeight.Bold,
```

Comment:
I took a look but in fact it's working as expected: when item is selected, font weight is null, so that the font weight of titleS is used, and titleS is bold. That's why it's always bold.

Anyway that's a good point, I remove this line since the DSM indicates that the font weight never changes.